onecodex/viz/__init__.py

```diff
@@ -120,7 +120,7 @@ def stderr_filter(line):
 
         altair_module.renderers.enable(
             "altair_saver",
-            fmts=["html", "svg"],
```

Comment:
@clausmith wanted to check with you on this one: a test was failing on this PR and Jai pointed this out to me as something Scott is also doing to get CI on his PR to pass. Do you know if it's okay to remove this here?

Reply:
I encountered a similar failing test, and spent a bit of time down a rabbit hole with googling and trial and error wrangling altair and notebook rendering deps, but couldn't resolve it with updating deps. In the end, the only thing I could get to work was changing this line to `fmts=["html"]`, which allowed the tests to pass.

The test that was failing for me was `test_html_report_generation()` in `tests/test_reports.py`, looking for the `vega-embed` text in the html doc.

@clausmith How can we update the PDF render functionality tests to make sure they cover the cases you're thinking of?

Reply:
fixed upstream and rebased